src/deadline/houdini_adaptor/HoudiniClient/houdini_handler.py

```diff
@@ -44,30 +44,17 @@ def set_node_settings(self, node):
         node_type = node.type().nameWithCategory().split("/")
         if node_type[0] == "Driver":
             if node_type[1] == "ifd":
-                # mantra render node
+                # Mantra render node
                 alfredProgress = node.parm("vm_alfprogress")
                 if alfredProgress is not None:
                     alfredProgress.set(1)
                     print("Enabled Alfred style progress")
-
-                verbosity = node.parm("vm_verbose")
-                if verbosity is not None:
-                    verbosity.set(3)
-                    print("Set verbosity to 3")
-
-            elif node_type[1] == "karma":
+            elif node_type[1] == "usdrender":
```

Comment:
Is this related to the comment, "Also noted some errors with node types that resulted in progress not being output by default in Karma."? Can we add those errors to the PR description so we know what's being fixed?

Karma definitely has verbosity settings, does only the inner usdrender have the ability to select alfred style progress?

Reply:
Yes! Updated the PR description to clarify that the node type for Karma is `"Driver/usdrender"` and fixed that everywhere